pkg/pillar/hypervisor/hypervisor.go

```diff
@@ -93,9 +94,18 @@ func BootTimeHypervisor() Hypervisor {
 // the advice of this function and always ask for the enabled one.
 func GetAvailableHypervisors() (all []string, enabled []string) {
 	all = hypervisorPriority
+	isHVTypeKube := base.IsHVTypeKube()
 	for _, v := range all {
 		if _, err := os.Stat(knownHypervisors[v].dom0handle); err == nil {
```

Comment:
With the above suggestion this would become
 if knownHypervisors[v].enabled() { 
     enabled = append(enabled, v)
 }

Reply:
I will submit a new PR for these suggestions.

Reply:
https://github.com/lf-edge/eve/pull/3892 submitted